### PR TITLE
Don't use client-only function on server

### PR DESCRIPTION
**Pull request by immibis**
Fixes my previous PR #5

### CODE REVIEW

**Comment by Shrimpstress** [comment]
Just implemented this in my latest commit, thanks though.

File: src/main/resources/computercraft.accesswidener [hunk 1]
```diff
@@ -20,3 +20,4 @@ accessible field net/minecraft/client/render/model/BakedQuad sprite Lnet/minecra
 accessible field net/minecraft/client/util/math/AffineTransformation translation Lnet/minecraft/client/util/math/Vector3f;
 accessible field net/minecraft/client/util/math/AffineTransformation scale Lnet/minecraft/client/util/math/Vector3f;
 accessible field net/minecraft/client/util/math/AffineTransformation rotation1 Lnet/minecraft/util/math/Quaternion;
+accessible field net/minecraft/block/entity/SignBlockEntity text [Lnet/minecraft/text/Text;
```
Comment (i509VCB) [inline src/main/resources/computercraft.accesswidener:23]: Accessor mixin do less brr

Comment (immibis) [reply src/main/resources/computercraft.accesswidener:23]: blame zundrel, i'm following the convention

Comment (Shrimpstress) [reply src/main/resources/computercraft.accesswidener:23]: accessor mixin many more line of code, more line of code = more file size. we are saving size here :crab:.

Comment (Ueaj-Kerman) [reply src/main/resources/computercraft.accesswidener:23]: i wil comit aw delet

Comment (Ueaj-Kerman) [reply src/main/resources/computercraft.accesswidener:23]: @i509VCB I have avenged you https://github.com/Zundrel/cc-tweaked-fabric/commit/1346a26179b9444aea4f1e5c6aff423370b5a366

**Pull request closed without merging**